src/renderer.ts

```diff
@@ -148,9 +160,16 @@ class TableRenderer {
   }
 
   toDot(): string {
+    const tooltip =
+      this.table.options.Note === undefined
+        ? ""
+        : `tooltip="${this.table.name}\\n${this.table.options.Note.replace(
+            '"',
+            '\\"'
+          )}";`;
     return `"${this.table.name}" [id="${
       this.table.name
-    }";label=<<TABLE BORDER="2" COLOR="#29235c" CELLBORDER="1" CELLSPACING="0" CELLPADDING="10" >
+    }";${tooltip}label=<<TABLE BORDER="2" COLOR="#29235c" CELLBORDER="1" CELLSPACING="0" CELLPADDING="10" >
```

Comment:
This seems to break the dot generator. When I run the output dot file I get the following error:

```
$cat .test-output/issue-9.dbml.dot | dot -Tsvg
Error: <stdin>: syntax error in line 10 near 'a'
```

Reply:
BTW, you could escape the quote using `JSON.stringify`, e.g.:

```typescript
JSON.stringify(`"hello" world`) \\ "\"hello\" world"
```

Reply:
Your `issue-9.dbml` has a note with multiple `"`, which was not supported => fixed.

Reply:
> BTW, you could escape the quote using `JSON.stringify`, e.g.:
> 
> ```ts
> JSON.stringify(`"hello" world`) // "\"hello\" world"
> ```

Yes, but `JSON.stringify` also adds starting and ending quotes, which we have to remove. What do you prefer?

Reply:
Uhm, good point... My current thinking goes towards using `stringify` and doing a `substring` afterwards taking the quotes out. That is, if the `stringify` escaping works well with graphviz.

In any case, could you also add a common `escape` function in the file? It seems that this logic was used in a couple of places already.

Reply:
I've created an  `escapeString` function (`escape` is already the name of a deprecated function in Javascript, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape), based on `JSON.stringify`.